### ISSUE TITLE
Better Ember Inspector integration

### ISSUE THREAD

**Issue by bendemboski** [adopted-ember-addons/ember-electron#425]
The API for installing extensions is a main process API, but we're doing it from the renderer processes via the Electron's `remote` module. This means that apps with `nodeIntegration` disabled will not be able to install the Ember Inspector, which is 100% as useful in non-`nodeIntegration` apps.

So I think we should do something like `electron-debug` does, and provide code that can be required from the main process to install the Electron Inspector, and add that code to `index.js` in the `ember-electron` blueprint/force template, so it would be something like:

```javascript
const { installEmberInspector } = require('ember-electron/extensions');
installEmberInspector();
```

Then it would be installed even with `nodeIntegration` disabled, and would allow users to opt out, and also learn about how install any other extensions they might want.

Open question is how to handle `devtron`. We could install it in the same manner, or point people to [electron-debug](https://github.com/sindresorhus/electron-debug), which I think is the most common way people install devtron.

**Comment by rwwagner90** [comment]
I had some issues with electron-debug. It would be nice to not require electron-debug to use it, but it's also not a huge deal either way.

**Comment by bendemboski** [comment]
@rwwagner90 whether `electron-debug` or not, my point is that devtron has nothing to do with Ember, and the problem of how to include it in an app is a problem that everybody with an Electron app has to solve, so I'd prefer to do it how everybody else does, whatever we decide that is (which is possibly doing nothing, and just providing info in an FAQ or something)

**Comment by rwwagner90** [comment]
Sure, totally get that point. I suppose I don't see the downside in keeping it included though?

**Comment by bendemboski** [comment]
Yeah, no big downside I don't think. Just maintenance/support cost of an increased "API surface area."

**Comment by rwwagner90** [comment]
I would vote we keep it, as I like to have more magic, like Ember provides out of the box. The more we can bolt on, to allow people to not really need to understand Electron and still hit the ground running, the better. That's just my opinion though, and I don't feel super strongly about it, so removing is fine too.

**Comment by bendemboski** [comment]
Yeah...I guess using the blueprint is a good compromise. It supplies the magic, but still allows users to opt out/customize. Let's leave it as an open question, and when this actually turns into a PR we can see how it comes together and discuss further.

**Comment by jacobq** [comment]
BTW, how does one control the version of `ember-inspector` that's used? Seems like it's used by `ember-electron` but then also added to the app, which makes me think we have a similar situation as we did for electron-forge in 
https://github.com/adopted-ember-addons/ember-electron/pull/432. Is that right?
(For context, I'm currently trying to use a forked version of ember-inspector and having to jump through more hoops than I expected https://registry.npmjs.org/@jacobq/ember-inspector/-/ember-inspector-3.13.0-SNAPSHOT.2019120900.tgz)

**Comment by bendemboski** [comment]
We look for it [here](https://github.com/adopted-ember-addons/ember-electron/blob/95f6a05312153686950288bc93c0d7730d532153/vendor/install-extensions.js#L33) from the renderer process, which should find the version installed in `electron-app` before it finds the version in the Ember app, per standard node dependency resolution rules. We install it there using the blueprint. We actually no longer need the version installed in the Ember app, I just forgot to remove it, but probably will as part of addressing this issue.

**Comment by jacobq** [comment]
> We actually no longer need the version installed in the Ember app, I just forgot to remove it, but probably will as part of addressing this issue.

Ah, thanks for mentioning that. I was wondering why it was there.

**Comment by bendemboski** [comment]
@rwwagner90 I believe you completely fixed this in #458 and #465 right? We should be able to close this now?

I know `ember-inspector` still doesn't seem to work reliably, but I think that's a separate issue, as it was the same with the old method of installing it as the new.

**Comment by rwwagner90** [comment]
Yeah, I think this can be closed 👍
